src/az390.java

```diff
@@ -2410,6 +2421,41 @@ else if (tz390.op_trace_type[index]==613)                          // #533
 
 
 
+/**
+ * Comparator for mnemonics to mimic EBCDIC collating sequence
+ * The mnemonic portion occupies the first 8 positions of the ReportEntry
+ * and contains only uppercase characters and digits.
+ * Digits should sort higher than alphabetic characters.
+ */
+public class ReportEntryComparator implements Comparator<String>                   // #568
+ {@Override                                                                        // #568
+  public int compare(String str1, String str2)                                     // #568
+   {// special case: entries consisting of a single space sort first!              // #568
+    if (str1.charAt(0) == ' ') return 1;  // blank entry always smaller            // #568
+    if (str2.charAt(0) == ' ') return -1; // blank entry always smaller            // #568
+    for (int i=0; i<7; i++)                                                        // #568
+     {char c1 = str1.charAt(i);                                                    // #568
+      char c2 = str2.charAt(i);                                                    // #568
+      if (c1 == c2) {} // continue on to next char                                 // #568
+      else // we found different characters                                        // #568
```

Comment:
Replace with "if (c1 != c2)" ???

Reply:
I did that to allow for the comments. I have put comments on a separate line, and followed your suggestion. 
Looks better this way, indeed. Thanks.